core/src/language/typescript.rs

```diff
@@ -110,13 +110,33 @@ impl Language for TypeScript {
 
     fn write_struct(&mut self, w: &mut dyn Write, rs: &RustStruct) -> std::io::Result<()> {
         self.write_comments(w, 0, &rs.comments)?;
+        let mut inheritance = "".to_string();
+        let mut count = 0;
+        for field in rs.fields.iter() {
+            if field.flattened {
+                let ts_ty = match self.format_type(&field.ty, rs.generic_types.as_slice()) {
```

Comment:
`rs.generic_types.as_slice()` can probably be `&rs.generic_types`

Reply:
done in c70b536